### ISSUE TITLE
Update to 'request' library breaks Apigeetool Node 0.10 compatibility

### ISSUE THREAD

**Issue by PizzaAficionado** [apigee/apigeetool-node#84]
The recent update to Request (2.82.0) specifies "hawk": "~6.0.2" in its dependencies.

Request has marked 2.82.0 as compatible with "request": "^2.63.0" as specified by this project's package.json

The recent updates to Hawk will not work with Node < 4. The update makes Apigeetool incompatible with Node 0.10.32

**Comment by ivan-novakov** [comment]
Yes, this somehow breaks our current continuous integration process. The latest "working" request version for me is v2.76.0.
